src/components/EventHandler.tsx

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
This file (and the next one `LibraryEntryPoint.tsx` are not *React.js components* so they should not be under `/src/components` folder, they should just be under `/src` folder. Also, since they do not contain any JSX/TSX contents, they should just be having an extension of `*.ts`, not `*.tsx`.